tests/drivers/uart/uart_mix_fifo_poll/testcase.yaml

```diff
@@ -1,7 +1,7 @@
 common:
   tags: drivers uart
```

Comment:
Thank you for your changes :)

Please add information about required GPIO dependency (depends_on: gpio).